kbl2vec/src/main/java/com/foursoft/harness/kbl2vec/transform/geometry/d2/CartesianPoint2DTransformer.java

```diff
@@ -0,0 +1,63 @@
+/*-
+ * ========================LICENSE_START=================================
+ * KBL to VEC Converter
+ * %%
+ * Copyright (C) 2025 4Soft GmbH
+ * %%
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ * =========================LICENSE_END==================================
+ */
+package com.foursoft.harness.kbl2vec.transform.geometry.d2;
+
+import com.foursoft.harness.kbl.v25.KblCartesianPoint;
+import com.foursoft.harness.kbl2vec.core.TransformationContext;
+import com.foursoft.harness.kbl2vec.core.TransformationResult;
+import com.foursoft.harness.kbl2vec.core.Transformer;
+import com.foursoft.harness.kbl2vec.transform.geometry.GeometryDimensionDetector;
+import com.foursoft.harness.vec.v2x.VecCartesianPoint2D;
+
+import java.util.List;
+
+import static com.foursoft.harness.kbl2vec.utils.ListUtils.getElementOrDefault;
+
+public class CartesianPoint2DTransformer implements Transformer<KblCartesianPoint, VecCartesianPoint2D> {
+
+    @Override
+    public TransformationResult<VecCartesianPoint2D> transform(final TransformationContext context,
+                                                               final KblCartesianPoint source) {
+        final VecCartesianPoint2D destination = new VecCartesianPoint2D();
+        final int DIMENSIONS = 2;
```

Comment:
Since this "Magic Number" seems the be repeating itself, make a it two `pulic static final` fields for 2D on 3D in the `GeometryDimensionDetector` and use them here all over the different classes.